src/core/ConfigManager.cpp

```diff
@@ -151,6 +153,7 @@ void ConfigManager::upgrade()
 
 	// Bump the version, now that we are upgraded
 	m_version = LMMS_VERSION;
+	// No need to bump the m_configVersion, because it gets bumped in the configVersion() method
```

Comment:
```suggestion
	m_configVersion = CONFIG_VERSION;
```

Reply:
To be clear, you would need to add `CONFIG_VERSION = 2;` somewhere in ConfigManager. Probably makes sense to put it in the header?